app/controllers/restaurants_controller.rb

```diff
@@ -1,13 +1,48 @@
 class RestaurantsController < ApplicationController
-  def index
-  end
+  	
+  	def index
+  		@restaurants = Restaurant.all
+  	end
+
+  	def show
+  		@restaurant = Restaurant.find(params[:id])
+  	end
+
+  	def new
+  		@restaurant = Restaurant.new
+  	end
+
+  	def create
+  		Restaurant.create(params.require(:restaurant)
+  			.permit(:name, :website, :created_at, :updated_at, 
+  			:hours, :low_price, :high_price))
+  		@restaurant = Restaurant.all
+  		redirect_to root_url
+  	end
+
+  	def edit
+   		@restaurant = Restaurant.find(params[:id])
+ 	end
+
+  	def update
+  		@restaurant = Restaurant.find(params[:id])
+  		@restaurant.update(params.require(:restaurant)
+  			.permit(:name, :website, :created_at,
+  			:updated_at, :hours, :low_price,
+  			:high_price))
+  		redirect_to root_url
+	end
+
+	def destroy
+		@restaurant = Restaurant.find(params[:id]).destroy!
+  		@restaurant.Restaurant.new
```

Comment:
I have no idea what line 38 is.

 

Heidi

 

From: Brian Gates [mailto:notifications@github.com] 
Sent: Tuesday, June 16, 2015 1:53 AM
To: TheIronYard--Orlando/ror_granular_app
Cc: Heidi L. Roude
Subject: Re: [ror_granular_app] ahhhhh, the domino errors and subsequent headaches (#14)

 

In app/controllers/restaurants_controller.rb <https://github.com/TheIronYard--Orlando/ror_granular_app/pull/14#discussion_r32492890> :

> +     def edit
> +            @restaurant = Restaurant.find(params[:id])
> +     end
> +
> +     def update
> +            @restaurant = Restaurant.find(params[:id])
> +            @restaurant.update(params.require(:restaurant)
> +                    .permit(:name, :website, :created_at,
> +                    :updated_at, :hours, :low_price,
> +                    :high_price))
> +            redirect_to root_url
> +     end
> +
> +     def destroy
> +            @restaurant = Restaurant.find(params[:id]).destroy!
> +            @restaurant.Restaurant.new

what does line 38 do?

—
Reply to this email directly or view it on GitHub <https://github.com/TheIronYard--Orlando/ror_granular_app/pull/14/files#r32492890> .  <https://github.com/notifications/beacon/AL4F_PoOOhCJxOE0OksqR_U7jwpZWCJgks5oT7FOgaJpZM4FD_cN.gif>